code-snippets.md

```diff
@@ -4,33 +4,48 @@ layout: default
 
 # Code Snippets
 
-Perhaps the biggest everyday productivity boost that you'll get in Preview 0.3
-is Code Snippets. These are pieces of code that expand when you type the code
-snippet (which appears in the IntelliSense list) and press TAB. 
+There are common pieces of code that you type every day such as create a plot or
+read a CSV file. To help make you more productive in your day-to-day work, we
+have added a number of `Code Snippets` to Visual Studio. A `Code Snippet` makes
+it easier for you to enter repeating code patterns. To insert one of those code
+patterns into your editor, you need:
+
+1. Type the abbreviated name of the Code Snippet (IntelliSense is available)
```

Comment:
That is markdown. It will number them automatically for you ... saves needing to renumber them manually :)